### DISCUSSION TITLE
PrimeVue 4, Nuxt 3 and Sakai Theme

### DISCUSSION THREAD

**Discussion by m0squito** [primefaces/community#3264] [PrimeVue]
I'm completely new to this front-end stack. I'm looking for help with setting up a basic Hello, World example for an app that uses Nuxt 3, PrimeVue 4 and the Sakai Theme. I'm all set with the first two steps, but am at a loss with how to install a theme that does not come with PrimeVue (like Aura). The Sakai Theme does not provide any help either, just points to Git repo with a demo. Any help would be greatly appreciated. 

_If this is not the right forum for this question - apologies - please point me in a better direction. Thanks._

**Comment by j0rgedev** [comment]
Hello @m0squito! I'm developing a nuxt Sakai vue template which is almost ready (80%). Let me know if you would like to have it

**Reply by m0squito** [reply]
Yes, absolutely. Thank you.

**Reply by j-t-black** [reply]
@j0rgedev Hi, I'm just starting out with Nuxt and would be interested as well! :)

**Reply by janderegg** [reply]
same for me - will you post here, when you're finished? Or is it already?

**Reply by j0rgedev** [reply]
Hello there! Please check my other comment where I linked the repo

**Reply by janderegg** [reply]
I saw that; but was thinking that was not the final one you‘ll be calling „Sakai Plus“?

**Reply by j0rgedev** [reply]
Oh, it's still under development. But I think I could make it public 🤔

**Reply by janderegg** [reply]
That would be great😊

**Reply by j0rgedev** [reply]
Got u! Unfortunately, I can't name it `Sakai Plus` since PrimeVue will launch a plus version as well [(see here)](https://x.com/cagataycivici/status/1871506992558412040). So, I was thinking about other good template names, I came up with Sokai, Sakana, Better Sakai, Sakai Nova, but I can't choose one 😥, any suggestions?

**Reply by jamurda** [reply]
How about Sakura? It's got a 'u' in it like nuxt, and it means cherry blossom in Japanese.

**Reply by janderegg** [reply]
+1 for Sakura - sounds great

**Comment by j0rgedev** [comment]
Hello! Sorry for the delay. As the template is not finished yet, I think you can use my personal sakai-nuxt template, which is just the starter template I use for my projects. I changed the default sidebar since I didn't like the default one, so I hope you like it as well! Keep in mind, this template is essentially a port of the Sakai Vue template, so it’s not fully Nuxt-compatible. You can find it here: https://github.com/j0rgedev/sakai-nuxt-starter. If you find any problem or have any suggestions, PRs are welcome 🫱🏻‍🫲🏻

Btw, the template I'm working on will be called Sakai Plus, and it'll have full nuxt compatible and TS support, ssr and spa templates, nuxt auth, nuxt layers, prisma, primevue forms, i18n and more! I’m aiming to have it ready by the end of the month 😁
